src/common/services/checkoutHelpers/checkout.service.js

```diff
@@ -0,0 +1,8 @@
+export function initializeRecaptcha () {
```

Comment:
Can we reduce the coupling between this method and the controllers that use it by passing in `$window` and `envService` as arguments?

Or we could make this a real Angular service and inject `$window` and `envService` as dependencies.